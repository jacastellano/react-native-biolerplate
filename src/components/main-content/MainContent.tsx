import * as React from 'react';
import { Container, Text } from "native-base";

interface Props {

}

export const MainContentComponent = (props: Props) => {
    return (
        <Container>
            <Text>Main Container</Text>
        </Container>
    )
}