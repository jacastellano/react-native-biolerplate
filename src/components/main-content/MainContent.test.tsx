import * as React from 'react';
import renderer from 'react-test-renderer';

import { MainContentComponent } from '../main-content/MainContent';

it('renders correctly', () => {
    const mainContent = renderer
        .create(<MainContentComponent />)
        .toJSON();
    expect(mainContent).toMatchSnapshot();
});