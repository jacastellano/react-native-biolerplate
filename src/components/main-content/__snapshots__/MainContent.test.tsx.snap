// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#000",
        "fontFamily": "System",
        "fontSize": 16,
      }
    }
    uppercase={false}
  >
    Main Container
  </Text>
</View>
`;
